manager/integration/tests/test_zone.py

```diff
@@ -14,8 +14,14 @@
 from random import randrange
 from test_scheduling import wait_new_replica_ready
 from common import get_version_api_client
+from common import CONDITION_STATUS_TRUE
+from common import wait_for_volume_condition_scheduled
+from common import wait_for_volume_delete
+from test_scheduling import SETTING_REPLICA_SOFT_ANTI_AFFINITY
 
 
+SETTING_ZONE_ANTI_AFFINITY = "replica-zone-soft-anti-affinity"
```

Comment:
Better to name the variable `SETTING_ZONE_SOFT_ANTI_AFFINITY`, it's more clear that it's hard or soft.

Reply:
Fixed, thanks